packages/flutter/test/material/date_picker_test.dart

```diff
@@ -417,4 +380,227 @@ void main() {
       expect(await date, isNull);
     });
   });
+
+  testWidgets('exports semantics', (WidgetTester tester) async {
+    final SemanticsTester semantics = new SemanticsTester(tester);
+    await preparePicker(tester, (Future<DateTime> date) async {
+      final TestSemantics expected = new TestSemantics(
+        children: <TestSemantics>[
+          new TestSemantics(
+            actions: 1,
```

Comment:
I find the integer here not very helpful. What is action 1? I'd prefer using the actual enums (here and below, also for flags).

With https://github.com/flutter/flutter/issues/12286 we should probably change the API here to directly take a list of enum values for actions and flags...

Reply:
Done.